src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1

```diff
@@ -233,8 +233,7 @@ class AutoBugLog : EventBase  {
                     }
                 }
                 else {
-                    $AssignedTo = $metaProviderObj.GetAssignee($ControlResults[0], $this.InvocationContext);
-                    $serviceId = $metaProviderObj.ServiceId;
+                    $AssignedTo = $ControlResults[0].ResourceContext.ResourceOwner
```

Comment:
Since we are not getting assignees from ST files, we can remove it from mandatory in autobuglogsvt.ps1
For org and proj controls we will still need the org mapping files, which we can handle here itself when logging bugs for these two resource types.